### DISCUSSION TITLE
it is possible to a dd custom Radios?

### DISCUSSION THREAD

**Discussion by alienator** [strawberrymusicplayer/strawberry#1439] [Q&A] [answered]
it is possible to a dd custom Radios?

**Comment by russelld00** [comment]
Adding internet radios is possible in Clementine, so code is available. 
Please add this to Strawberry, thanks in advance!

**Comment by jonaski** [comment, marked as answer]
Currently not possible, suggest it on: https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
